backend_addon/{{ cookiecutter.__folder_name }}/src/{{ cookiecutter.__package_namespace }}/{{ cookiecutter.__package_name }}/dependencies.zcml

```diff
@@ -2,8 +2,8 @@
 
   <!-- List all packages you depend here -->
   {%- if cookiecutter.__feature_headless == '1' %}
-  <include package="plone.volto" />
   <include package="plone.restapi" />
```

Comment:
`plone.volto` pulls in `plone.restapi` anyway, no need to actively include it here AFAIK.